docs/reference/ml/df-analytics/apis/delete-dfanalytics.asciidoc

```diff
@@ -32,6 +32,13 @@ information, see <<security-privileges>> and <<built-in-roles>>.
 (Required, string)
 include::{docdir}/ml/ml-shared.asciidoc[tag=job-id-data-frame-analytics]
 
+[[ml-delete-dfanalytics-query-params]]
+==== {api-query-params-title}
+
+`force`::
+  (Optional, boolean) Use to forcefully delete a job that is not stopped; this method is
```

Comment:
#nitpicking
If I understand correctly then in this case "forcefully" means "deleting the job without stopping it beforehand". If my understanding is correct, then I think "forcefully" is redundant. 

```suggestion
  (Optional, boolean) Deletes a job that is not stopped; this method is
```